packages/engine/lib/execution/src/runner/python/batch.py

```diff
@@ -289,4 +290,6 @@ def free(self):
         #       (Call _free_rust_static_meta, _free_rust_dynamic_meta, unload_shared_mem here?)
```

Comment:
Is this TODO fully addressed, or do you just want to remove `unload_shared_mem` from the comment.

Also do we want to consider the other TODO about making it `__del__` or not?

Reply:
Great find! I agree.

Unfortunately a lot of the runner's code (both runners) are written for fairly outdated specs of their languages so there's probably quite a lot of this